package-lock.json

```diff
@@ -63,7 +64,7 @@
         "eslint-plugin-jsx-a11y": "^6.5.1",
         "eslint-plugin-react": "^7.27.1",
         "husky": "^7.0.4",
-        "jsdoc": "^3.6.7",
+        "jsdoc": "^4.0.2",
```

Comment:
This change should be un-done automatically when you revert the change in package.json as Bennett requested. 

Other changes marked "revert" below should also go away at the same time. 
 But check that they do.  I think the ones marked "keep" will stay.